corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -53,14 +53,14 @@
     <select>
 
     <% } else if (input == "daterange") { %>
-    <input type="text" class="daterange query-field form-control" value="<%- value %>">
+    <input aria-label="<%- text ? text : "" %> daterange" type="text" class="daterange query-field form-control" value="<%- value %>">
```

Comment:
Does the screenreader know how to read "daterange" (without a space)?

Reply:
You mean "daterange" vs "date range", correct? That's an interesting question. It does read it correctly, but if I hadn't already tested already I would wonder the same thing.

Reply:
Yup exactly and it works anyways so 👍